test/e2e/flask/solana/common-solana.ts

```diff
@@ -1696,15 +1696,16 @@ export async function withSolanaAccountSnap(
     },
     async ({ driver, mockServer }: { driver: Driver; mockServer: Mockttp }) => {
       await loginWithoutBalanceValidation(driver);
-      const headerComponen = new HeaderNavbar(driver);
-      await headerComponen.openAccountMenu();
+      const headerComponent = new HeaderNavbar(driver);
+      await headerComponent.openAccountMenu();
       const accountListPage = new AccountListPage(driver);
       if (!importAccount) {
         await accountListPage.addAccount({
           accountType: ACCOUNT_TYPE.Solana,
           accountName: 'Solana 1',
         });
       }
+      await headerComponent.check_accountLabel('Solana 1');
```

Comment:
this ensures that we have loaded the solana account in the wallet before proceeding to the next step